src/js/node/net.ts

```diff
@@ -1470,7 +1452,7 @@ Server.prototype.listen = function listen(port, hostname, onListen) {
         tls.rejectUnauthorized = false;
       }
     } else {
-      options.InternalSocketClass = SocketClass;
+      options.InternalSocketClass = Socket;
```

Comment:
Is this a real thing? InternalSocketClass? If this was ment as an internal api, can we use a symbol?